test/ng/compileSpec.js

```diff
@@ -2113,6 +2113,7 @@ describe('$compile', function() {
         var iscope;
 
         beforeEach(module(function() {
+          var fakeModule = {name: 'fakeModule'};
```

Comment:
Faking out a module to pass to the directive function so the test will show the result of passing a module.